certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3294 @@
+	{
+	  "swagger": "2.0",
+	  "info": {
+		"version": "1.0.4",
+		"title": "VirusTotal",
+		"description": "Virus Total is an online service that analyzes suspicious files and URLs to detect types of malware and malicious content using antivirus engines and website scanners. It provides an API that allows users to access the information generated by VirusTotal.",
+		"contact": {
+		  "name": "Microsoft",
+		  "url": "https://azure.microsoft.com/",
+		  "email": "azuresentinel@microsoft.com"
+		},
+		"x-ms-api-annotation": {
+		  "status": "Preview"
+		}
+	  },
+	  "host": "www.virustotal.com",
+	  "basePath": "/",
+	  "schemes": [
+		"https"
+	  ],
+	  "consumes": [],
+	  "produces": [
+		"application/json"
+	  ],
+	  "paths": {
+		"/api/v3/ip_addresses/{ip}": {
+		  "get": {
+			"summary": "IP scan report",
+			"description": "Retrieve information about an IP address",
+			"operationId": "VirusTotalGetIpScan",
```

Comment:
@ytognder The apiDefinition.swagger.json file is outdated - Please update the latest one you have uploaded in the certification portal

 "/api/v3/ip_addresses/{ip}": {
            "get": {
                "description": "Retrieve information about an IP address",
                "operationId": "VirusTotalGetIpScan",
                "parameters": [
                    {
                        "description": "IP to verify",
                        "in": "path",
                        "name": "ip",
                        "required": true,
                        "type": "string",
                        "x-ms-summary": "IP Address",
                        "x-ms-url-encoding": "single"
                    }
                ],
                "responses": {
                    "200": {
                        "description": "Ok",
                        "schema": {
                            "$ref": "#/definitions/ipResult"
                        }
                    }
                },
                "summary": "IP scan report"
            }
        },

should be
"/api/v3/ip_addresses/{ip}": {
            "get": {
                "description": "Scan IP address to get full report",
                "operationId": "VirusTotalGetIpScanV3",
                "parameters": [
                    {
                        "description": "Ip to verify",
                        "in": "path",
                        "name": "ip",
                        "required": true,
                        "type": "string",
                        "x-ms-summary": "IP Address",
                        "x-ms-url-encoding": "single"
                    }
                ],
                "responses": {
                    "200": {
                        "description": "Ok",
                        "schema": {
                            "$ref": "#/definitions/ipResult"
                        }
                    }
                },
                "summary": "Ip scan report V3"
            }
        },